infrastructure/variables.tf

```diff
@@ -31,9 +31,16 @@ variable "tf_state_bucket" {
   description = "The name of the S3 bucket where the state is stored"
 }
 
+variable "raw_data_bucket" {
+  type        = string
+  default     = "landgriffon-raw-data"
+  description = "The name of the s3 buckets where where raw data will be stored"
```

Comment:
review the description please

Reply:
Fixed [here](https://github.com/Vizzuality/landgriffon/pull/10/commits/714a6d7cc7974b7aa38f326f7dfd6af112c8c399)